CONTRIBUTING.md

```diff
@@ -145,7 +145,14 @@ The process for a code change and pull request you should follow:
 "feature/###" or "fix/###". For more information see the Git branching guideline.
 1. Make changes, compile, and test thoroughly. Ensure any install or build dependencies are removed before the end of the layer when doing a build. Code style should match existing style and conventions, and changes should be focused on the topic the pull request will be addressed. For more information see the style guide.
 1. Push commits to your fork.
-1. Create a Github pull request from your topic branch.
+1. Create a Github pull request from your topic branch, following the steps below:
+    1. Name your PR with "{Feature or EPIC name}" + ": " + "{title for this particular PR}": e.g., "Step-up transformer tap changer support: updated ranking"
```

Comment:
what if there's no associated feature? e.g. https://github.com/PowerGridModel/power-grid-model/pull/825 (`Fix nightly build`). Would it be something along the lines of `Nightly build: fix`?

Reply:
I really don't think a general guideline should include all edge cases but in this case I would say `Nightly build` fits in the category of feature or epic and should indeed go along those lines. I will mention that "{Feature or EPIC name}: " is not as strictly required when there is no bigger feature that the PR is belonging to.